src/app/chatbot-page/interview/interview.component.scss

```diff
@@ -1,249 +1,225 @@
+  }
+  100% {
+    opacity: 1;
+    transform: translateY(0);
+  }
 }
 
 .answers {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    max-width: 50%;
-    width: 50%;
-    min-height: 100%;
-
- 
-
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  max-width: 50%;
+  width: 50%;
+  min-height: 100%;
 }
 
 .answers div {
-    width: 100%;
-}
-.MultipleChoiceAnswers {
-    display: flex;
-    flex-wrap: wrap;
-    gap: 10px;
-    justify-content: left;
-    
+  width: 100%;
+
 }
 
+.MultipleChoiceAnswers {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  justify-content: left;
 
+}
 
-mat-button-toggle-group { //Rahmen in dem alle Antwort buttons sind;
-    flex-wrap: wrap;
- 
-    gap: 10px;
-    outline: none;
-    padding: 20px; /* Fügt innen Abstand hinzu und vergrößert somit den Rahmen */
-    margin: 20px;  /* Fügt außen Abstand hinzu und vergrößert den Abstand um den Rahmen herum */
-    box-shadow: rgba(0, 0, 0, 0.07) 0px 0px 25px 5px;
-    border-radius: 30px; /* Fügt abgerundete Ecken hinzu */
-    overflow-y: auto;
+mat-button-toggle-group {
+  flex-wrap: wrap;
+  gap: 10px;
+  outline: none;
+  padding: 20px;
+  margin: 20px;
+  box-shadow: rgba(0, 0, 0, 0.07) 0px 0px 25px 5px;
+  border-radius: 30px;
+  overflow-y: auto;
 }
-//nur für die Auswahl ,,Englisch"
-.mat-button-toggle { 
-    background-color: white;
-    border-radius: 60px;
-    font-family: "Rubik", sans-serif;
-    padding: 0px 0px;   /* Du kannst die Polsterung hier hinzufügen oder ändern */
-    margin: 0px 12px 12px 0px;  
-    border: 1px solid black;  /* Du kannst die Rahmenfarbe und -breite hier ändern */
-    font-size:20px;
-    transition: background-color 0.3s ease, color 0.3s ease, border 0.3s ease;
+
+.mat-button-toggle {
+  background-color: white;
+  border-radius: 60px;
+  font-family: "Rubik", sans-serif;
+  padding: 0px 0px;
+  margin: 0px 12px 12px 0px;
+  border: 1px solid black;
+  font-size: 20px;
+  transition: background-color 0.3s ease, color 0.3s ease, border 0.3s ease;
 }
 
 .mat-button-toggle-checked {
-    background-color: var(--color-primary) !important;
-    font-family: "Rubik", sans-serif;
-    margin: 0px 12px 12px 0px;  
-    font-size:20px;
-    color:white !important;
-    border: none !important;
-    border-radius: 60px;
-    filter:contrast(1.5);
-    
+  background-color: var(--color-primary) !important;
+  font-family: "Rubik", sans-serif;
+  margin: 0px 12px 12px 0px;
+  font-size: 20px;
+  color: white !important;
+  border: none !important;
+  border-radius: 60px;
+  filter: contrast(1.5);
 }
 
 .mat-button-toggle:hover {
-    background-color:  var(--color-primary-lighter)!important; // Primärfarbe mit 80% Deckkraft für den Hover-Effekt
-    color: white !important;
-        border:none !important;
+  background-color: var(--color-primary-lighter) !important;
+  color: white !important;
+  border: none !important;
 }
 
-//Aussehen fuer jeden einzelnden Antwort button;
```

Comment:
Same here, I would translate this comment to english and keep it there in case we need to do a quick change to the buttons later